### ISSUE TITLE
Other trees get collapsed (when set not to) when dragging and hovering a tab on top of any tab

### ISSUE THREAD

**Issue by azizLIGHT** [piroor/treestyletab#2224]
## Short description
All other trees collapse when dragging a tab on top of other tabs, except for the tree where the tab is being dragged to. 

## Steps to reproduce

 1. Start Firefox with clean profile.
 2. Install TST.
 3. On TST welcome page, check option for "Don't expand collapsed tree and skip collapsed descendants, while switching focus of tabs via keyboard shortcuts (requires script injection to webpages for detection of keyboard operations)" 
 
4. in TST settings -> Tree Behavior section:
- uncheck option for "When a new tree appears, collapse others automatically"
- uncheck option for "When a tab gets focus, expand its tree and collapse others automatically"
- uncheck option for "Except focus moving caused by closing of the current tab"

5. Create Tree Structure:
```
Parent 1
	Child 1
Parent 2
	Child 2
Parent 3
	Child 3
Parent 4
Lone Tab 1
```
6. Try any combination of the following repeatable behaviors:
Drag Lone Tab 1 by clicking mouse button down on it and then not letting go of mouse button, then move mouse to drag Lone Tab 1 on top of: 
- Parent 3 or Child 3. Observe Parent 1 and Parent 2 tree collapse after 1-2 seconds
- Parent 1 or Child 1. Observe Parent 2 and Parent 3 trees collapse after 1-2 seconds.
- Parent 2 or Child 2. Observe Parent 1 and Parent 3 trees collapse after 1-2 seconds.
- Parent 4. Observe Parent 1, Parent 2, and Parent 3 trees collapse after 1-2 seconds.

There may be other situations where this is repeatable as well, that I have not tested.
FYI, behavior does not happen dragging and hovering between Parent or Child, or between Parents, or above Parents or below Parents, IE: not on top of a tab

## Expected result
Expecting all tree structure to stay expanded when dragging and hovering Lone Tab 1 to any location for any time duration

## Actual result
Tree structure collapses for all trees except where Lone Tab 1 is being dragged

## Environment

 * Platform (OS): Windows 7
 * Version of Firefox: 66.0.2 (64-bit)
 * Version (or revision) of Tree Style Tab: 3.0.5 April 10, 2019

## Screenshots/Video
![treestyletab-collapsing](https://user-images.githubusercontent.com/8013867/55885581-34b52280-5b78-11e9-8d90-91d345235e05.gif)
![image](https://user-images.githubusercontent.com/8013867/55885728-734add00-5b78-11e9-883b-15c2b8df22b6.png)

## Comments
TST needs a setting that says do not collapse/expand at all, if possible, if it makes sense. If it can be respected, then it would provide predictability to the user. As it is now, this collapsing behavior can cause visual confusion and disruption of user workflow when the user gets lost where they were in the tree structure. It can be especially disorienting if their tree structures are very complex.

In the confusion, one may accidentally drop the tab somewhere and not be able to find it, if the tree structure is very complex and lots of parents are collapsed, causing a large vertical displacement in position of all the tabs when this unasked-for collapsing behavior occurs.

An example in a complex tree shows the large vertical displacement and likely confusion:
![treestyletab-collapsing-complex](https://user-images.githubusercontent.com/8013867/55886879-89599d00-5b7a-11e9-8fd4-7ba833b23bf2.gif)

**Comment by masterseoonline** [comment]
same here its very annoying, ( i have thousand tab)
also when i enable debug mode, the option page disapear completly
is 3.0.4 version have this issue also?
i want to downgrade

**Comment by piroor** [comment]
Thanks, this odd behavior should be fixed with the commit 60e5e81. Now TST just ignores already expanded tree on such cases.

**Comment by irvinm** [comment]
Whew! I thought I was losing my mind!  Thanks @azizLIGHT for reporting it and of course @piroor for a quick fix!

**Comment by azizLIGHT** [comment]
Fixed for me on 3.0.6. Thanks
